### ISSUE TITLE
Corrupted project

### ISSUE THREAD

**Issue by sfederici** [TurboWarp/sb3fix#30]
When I try to open [this project](https://github.com/user-attachments/files/17099560/obiettivi-variabili_.Pia.Valeria67592.sb3.zip) in TW I get this error

<img width="301" alt="immagine" src="https://github.com/user-attachments/assets/2c6f5623-cb0b-432e-ac32-bc00b16951a4">

Even SB3fix can't solve the problem.

Thanks in advance

**Comment by GarboMuffin** [comment]
I put this into `zip -FF` and got this for you: https://static.muffin.ink/sb3fix-30.sb3

it does load and the scripts are there, but some images are missing...

Was this project saved using the desktop app or website?

**Comment by sfederici** [comment]
> I put this into `zip -FF` and got this for you: https://static.muffin.ink/sb3fix-30.sb3

Thanks. When I click the above link I get a "Error 500 There was an internal error We were not able to find this object Cloudflare was unable to retrieve the object at this time. Please refresh or try again later."

> it does load and the scripts are there, but some images are missing...

The important part is the scripts, thanks.

> Was this project saved using the desktop app or website?

Desktop version. But, as you know, my customized version. Even if, in almost a year, this was the very first time I got a problem.

**Comment by GarboMuffin** [comment]
this link should work: [sb3fix-30.sb3.zip](https://github.com/user-attachments/files/17101048/sb3fix-30.sb3.zip)

**Comment by GarboMuffin** [comment]
also, do check the restore points if you havent already, the full version might still be in there if you havent turned them off

<img width="280" alt="image" src="https://github.com/user-attachments/assets/60b3b0aa-8a59-4f3f-b39f-2fab5f62780e">

**Comment by sfederici** [comment]
> this link should work: [sb3fix-30.sb3.zip](https://github.com/user-attachments/files/17101048/sb3fix-30.sb3.zip)

Yes, it does

**Comment by sfederici** [comment]
> also, do check the restore points if you havent already, the full version might still be in there if you havent turned them off
> <img alt="image" width="280" src="https://private-user-images.githubusercontent.com/33787854/369969568-60b3b0aa-8a59-4f3f-b39f-2fab5f62780e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxMTAzODAsIm5iZiI6MTcyNzExMDA4MCwicGF0aCI6Ii8zMzc4Nzg1NC8zNjk5Njk1NjgtNjBiM2IwYWEtOGE1OS00ZjNmLWIzOWYtMmZhYjVmNjI3ODBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIzVDE2NDgwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxM2Y5ZDNlZjRkMGY3NTRhYjYwNmU5M2FjODU1YjkzY2IyODg0NzFiYTA5NzNmOTYxNzk4YWY4N2MxYzE2YTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.EScAoVp6MaQAPo3adF_Fa8PxAUQC13-MvbKLgdkToZs">

Yes, I know this feature. Thanks. Unfortunately it was created for today's exam by a student on a lab PC, and they are reset at the end of each session.
